### PR TITLE
Feature/add task manager

### PR DESCRIPTION
**Pull request by BarbosaJackson**

### CODE REVIEW

File: nexus/task_managers/file_manager/celery_file_manager.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import os
```
Comment (zMardone) [inline nexus/task_managers/file_manager/celery_file_manager.py:1]: Flake8: Not used celery and settings imports

File: nexus/task_managers/tasks.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from nexus.celery import app
+
+from nexus.task_managers.file_database.s3_file_database import s3FileDatabase
```
Comment (zMardone) [inline nexus/task_managers/tasks.py:3]: Flake8: Not used import

File: nexus/task_managers/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (zMardone) [inline nexus/task_managers/tests.py:1]: Flake8, not used import. You may also delete this file and add it later with the corresponding unittests.

File: nexus/task_managers/file_database/sentenx_file_database.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import requests
+from django.conf import settings
+
+from nexus.task_managers.models import TaskManager
+
+
+class SentenXFileDataBase:
+    def __init__(self):
```
Comment (zMardone) [inline nexus/task_managers/file_database/sentenx_file_database.py:8]: Check if this __init__ function is needed

File: nexus/intelligences/api/views.py [hunk 5]
```diff
@@ -236,3 +248,43 @@ def destroy(self, request):
         return Response(
             status=status.HTTP_204_NO_CONTENT
         )
+
+
+class ContentBaseFileViewset(ModelViewSet):
```
Comment (zMardone) [inline nexus/intelligences/api/views.py:253]: Change ModelViewset to retrive, create and list mixin. Or add usecases for the other mixins as well to use model mixin.

File: nexus/task_managers/tasks.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+from nexus.task_managers.file_database.sentenx_file_database import SentenXFileDataBase
+from nexus.usecases.task_managers.celery_task_manager import CeleryTaskManagerUseCase
+from nexus.task_managers.models import ContentBaseFileTaskManager
+
+@app.task
+def add_file(task_manager_uuid):
+    try:
+        task_manager = CeleryTaskManagerUseCase().get_task_manager_by_uuid(task_uuid=task_manager_uuid)
+        task_manager.update_status(ContentBaseFileTaskManager.STATUS_LOADING)
+        print(f"[ ADD FILE ] - updated status from task {task_manager.uuid}")
```
Comment (AlisoSouza) [inline nexus/task_managers/tasks.py:13]: remove the print statements in the code

**Review by AlisoSouza — APPROVED** [review]

**Pull request merged**